src/docs/product/discover-queries/query-builder/query-equations.mdx

```diff
@@ -0,0 +1,141 @@
+- [A custom Apdex score](/#calculate-a-custom-apdex)
+- [The average non-database times](#find-largest-average-non-database-times)
+
+To add an equation to a query:
+
+1. Click the "Columns" button to open the column editor.
+2. If they aren't already included in your query, add the columns you need for your equation.
+3. Click "Add an Equation".
+4. Enter your equation by selecting columns, entering numbers (if needed), and adding operators.
+5. Click "Apply".
+
+## Guidelines for Equations
+
+Equations must:
+
+- Contain at least one field or function
+- Contain at least one operator
+- Have explicit operators. For example, `a(a+b)` is invalid; instead, you should enter `a * (a + b)`
+
+Equations cannot:
+
+- Contain a mix of fields or functions
+- Contain exponents which aren't supported
+
+Equations will follow expected order of operations, resolving brackets first, division and multiplication next followed by addition and subtraction. Equations can also include negative numbers.
+
+Equations can only operate on columns and functions that are numbers, which includes:
+
+- Functions:
+    - count
+    - count_unique
+    - count_if
+    - failure_count
+    - avg
+    - sum
+    - percentiles (p50, p75 etc.)
+    - apdex
+    - user_misery
+    - eps() and epm()
+- Fields
+    - transaction.duration
+    - measurements.fp
+    - measurements.fcp
+    - measurements.lcp
+    - measurements.fid
+    - measurements.cls
+    - measurements.ttfb
+    - spans.browser
+    - spans.http
+    - spans.db
+    - spans.resource
+
+## Example Equations
+
+### Calculate a status percentage
+
+Get the percentage of transactions that completed within a threshold
+
+1. Create a new query with the following three columns:
+    - Column 1:
+        - Function  `count_if`
+        - Field `transaction.duration`
+        - Condition `is less than or equal to`
+        - Set final parameter to a threshold (like 300)
+    - Column 2:
+        - Function `count`
+    - Column 3:
+        - Field `transaction`
+2. Add an equation and enter  `count_if(transaction.duration,lessOrEquals,300) / count() * 100` in the equation field.
+
+### Calculate a custom Apdex
+
+Apdex is defined as (satisfied + tolerable*0.5 + frustrated*0)/total. If you wanted to change the the weight of tolerable to 0.2, you would create the following query and equation.
+
+1. Create a new query with the following four columns:
+    - Column 1:
+        - Function `count_if`
+        - Field  `transaction.duration`
+        - Condition `is less than or equal to`
+        - Set the final parameter to a Satisfied threshold (like 300)
+    - Column 2:
+        - Function `count_if`
+        - Field `transaction.duration`
+        - Condition `is less than or equal to`
+        - Set the final parameter to a Tolerable threshold (like 1200)
+    - Column 3:
+        - Function `Count()`
+    - Column 4:
+        - Field  `transaction`
+2. Add an equation and enter the following in the equation field:
```

Comment:
"Add an equation" -> can we change this to "Click the 'Add Equation' button" in the instructions?

Reply:
We've already spelled out the steps previously and we're not spelling out every step in these examples, so it would make for a non-parallel structure